build.yaml

```diff
@@ -427,9 +427,9 @@ common_cli:
 # flocker.node.functional is hanging, so we don't run it
 job_type:
   run_trial:
-  # http://build.clusterhq.com/builders/flocker-centos-7
+    # http://build.clusterhq.com/builders/flocker-centos-7
     run_trial_on_AWS_CentOS_7:
-      on_nodes_with_labels: 'aws-centos-7-T2Small'
+      on_nodes_with_labels: 'aws-centos-7-SELinux-T2Small'
```

Comment:
Why is this particular configuration detail in the node label?
Surely there are many configuration details which are not in the labels.
Is it because the labels must be differentiated from the last set of AMI labels? If so, maybe we should come up with a system (dates? version numbers?) for labelling.

This can be left for a later date but please reply letting me know your thoughts.